samples/Mvc.RenderViewToString/EmailGraphHelper.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Mvc.RenderViewToString
+{
+    public class EmailReportHelper
```

Comment:
> This is simply present to demonstrate `@inject`.

I'm struggling with this because views make extensive use of the `@inject` infrastructure without users entering the keyword. Suggest the sample would be clearer without this.

If it stays, would prefer new names for this and `RazorViewHelper`.